### PR TITLE
[inspector] Add loading view view

### PR DESCRIPTION
**Pull request by jasonLaster**
This adds a sweet spinner when you select a view that's not yet fetched. If you've used the inspector much since the lazyWorker was introduced, you'll have noticed that there's a decent 1 second or so lag time for decently large apps. not a big deal, but it seems like a bug not to show something in the UI...

> btw trying to also bump the line count post sam's slaughter

### CODE REVIEW

File: recorder/sandbox.rb [hunk 1]
```diff
@@ -46,6 +46,16 @@
...
 
+get "/extension/fonts/fontawesome-webfont.ttf?v=4.2.0" do
+  content_type 'application/font-woff'
+  File.read(File.join('..', 'extension/fonts/fontawesome-webfont.ttf'))
+end
+
+get "/extension/fonts/fontawesome-webfont.woff?v=4.2.0" do
+  content_type 'application/font-woff'
+  File.read(File.join('..', 'extension/fonts/fontawesome-webfont.woff'))
+end
```
Comment (jasonLaster) [inline recorder/sandbox.rb]: not fun :/

File: extension/templates/devTools/ui/loading.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+<div style="margin:20px;" class="panel panel-warning">
```
Comment (paulfalgout) [inline extension/templates/devTools/ui/loading.html]: maybe not now, but we should get some of these margins and other common one offs in predefined classes @nerdpruitt ping

File: recorder/sandbox.rb (hunk 1 above)
Comment (paulfalgout) [inline recorder/sandbox.rb]: What is going on here?  should we just make a custom font?  Could be small and clean and we'd only be concerned with the one file type.

File: extension/templates/devTools/ui/loading.html (hunk 2 above)
Comment (jasonLaster) [inline extension/templates/devTools/ui/loading.html]: Oops. Yea, sloppy.

File: recorder/sandbox.rb (hunk 1 above)
Comment (jasonLaster) [inline recorder/sandbox.rb]: I'm not sure it'd help too much. Like download size is not a huge issue. With that said I included both cause I don't have a strong handle on font sources, so happy for some more expertise.

**Pull request merged**